test/run-tests

```diff
@@ -750,7 +778,8 @@ def handle_task(gh, args, config, task, dry_run=False):
                     description += ": Error uploading results"
 
             # FIXME: workdir might be kept when python crashes
-            shutil.rmtree(workdir)
+            if os.environ.get("TEST_HARNESS_KEEP_RESULTS", "false") == "false":
```

Comment:
It is probably cleaner to make this a command-line flag so all config options are in the same place.

Reply:
fixed in https://github.com/linux-system-roles/test-harness/pull/87/commits/a1f7e36ec876eb927bb6ddcd25ba619b13dc88c0